### PR TITLE
env configuration

### PR DESCRIPTION
**Pull request by colman-hartinger**
Essentially there are 2 configurations here 

1: running in node
node can read from a .env file so this is straightforward

(implemented and tested)

2. running in a browser
the webpack file will read in the version from .env file for browser use

(implemented and not tested)

### CODE REVIEW

**Comment by colman-hartinger** [comment]
I think having a .env file is a cleaner solution than trying to read in the package.json or creating [environmentVariables.js](https://stackoverflow.com/questions/36962601/node-js-how-to-use-environment-variables-in-json-file) file but I'm open to going either of those routes as well

**Review by bradymadams — CHANGES_REQUESTED** [review]
This looks good to me, but can we include the revision number in the .env file?

**Comment by colman-hartinger** [comment]
@bradymadams revision number is now included in .env file

File: src/api.ts [hunk 1]
```diff
@@ -84,10 +89,9 @@ export class API {
   }
 
   static get version() {
-    return '21.0';
-    // return (typeof THOR_VERSION === 'undefined' ? '21.0' : THOR_VERSION);
+    return this.version
   }
-
+
```
Comment (bradymadams) [inline src/api.ts:94]: Added whitespace here and line 83.

**Review by bradymadams — COMMENTED** [review]
Can we not do?

```
THOR_VERSION = 21.0.1
```

The env variable should be a string, so I think that should be fine, but if they need to be separate let's drop the `_NUMBER` suffix off the revision variable name.

Oh ya we can definitely do that, I thought that when you were asking to add a revision number you wanted a separate variable

**Review by bradymadams — APPROVED** [review]

**Pull request merged**